tooling/template-base/components/LearnerAccess/ListDisplayDropDown.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  HydratedContentItem,
+  useUserCourseCompletionProgressQuery
+} from '@thoughtindustries/content';
+import React from 'react';
+interface ContentUiProps {
+  item: HydratedContentItem;
+  index?: number;
+}
+
+const LearnerAccessListDisplayDropDown = ({ item }: ContentUiProps) => {
+  const { data } = useUserCourseCompletionProgressQuery({
+    variables: {
+      id: item.id
+    }
+  });
+  return (
+    <>
+      <hr className="bg-gray-500"></hr>
+      <div className="flex flex-col bg-slate-50 p-6 space-y-6 border-l-4 border-blue-700">
+        {/* article and duration */}
+        <div className="flex flex-row">
+          <div className="py-1 px-3 rounded-full bg-green-200">
+            <div className="text-sm font-semibold">{item.contentTypeLabel}</div>
+          </div>
+          <div className="flex items-center text-gray-500 px-4">|</div>
+          <div className="flex items-center text-gray-500 font-sm font-semibold">30 mins</div>
+        </div>
+        {/* course information */}
+        <div className="flex flex-row justify-between gap-4">
+          {data?.UserCourseCompletionProgress?.map(
+            item =>
+              item?.type !== 'coursePercentViewed' && (
+                <div className="flex flex-grow pl-6 py-6 bg-white rounded-md">
+                  <div className="flex flex-col">
+                    <div className="text-sm text-gray-500">
+                      {item?.type?.replace(/([A-Z]+)/g, ' $1').replace(/([A-Z][a-z])/g, ' $1')}
+                    </div>
+                    <div className="text-3xl font-bold pt-2">
+                      {item.completed?.length}/{item.required?.length}
```

Comment:
Let's test for these values before displaying them here, if one of them is missing it could look malformed.

Reply:
test added